subsys/net/l2/ethernet/bridge/bridge.c

```diff
@@ -255,8 +255,8 @@ static void bridge_iface_init(struct net_if *iface)
 	ctx->iface = iface;
 
 	net_if_flag_set(iface, NET_IF_NO_AUTO_START);
-	net_if_flag_clear(iface, NET_IF_IPV4);
-	net_if_flag_clear(iface, NET_IF_IPV6);
+	net_if_flag_set(iface, NET_IF_IPV4);
+	net_if_flag_set(iface, NET_IF_IPV6);
```

Comment:
Why do we need to do this as bridging is done in L2?

Reply:
Fair enough. I would suggest adding to the commit message the reasoning too, new it states the obvious thing which we can see from the changes but does not answer the question "why this change is done".